### DISCUSSION TITLE
Can't change language

### DISCUSSION THREAD

**Discussion by cleysonmayrink** [mozilla/pdf.js#17744] [Q&A]
How i can change the language? i download the language file from [https://github.com/mozilla/pdf.js/tree/master/l10n](url) (its a .ftl file)
and a ref like this:

`<link href="static/components/vue-pdf-app/locale.ftl" rel="resource" type="application/l10n" />`

but still dont work, 

can someone help me please!?

**Comment by JOSTInternational** [comment]
I also have this issue with V4.0.189. Can a language change solution be ported to earlier versions of pdf.js? I have not updated to the latest version of pdf.js due to complexity of changes. Google translate API works very well when prompted. I do not know what property this is modifying, and I cannot get consistent google translate API pop-ups because my development locale is in USA.
